conf/pipeline/sarek/munin.config

```diff
@@ -0,0 +1,24 @@
+// Profile config names for nf-core/configs
+
+params {
+  // Specific nf-core/configs params
+  config_profile_contact = 'Maxime Garcia (@MaxUlysse)'
+  config_profile_description = 'nf-core/sarek MUNIN profile provided by nf-core/configs'
+
+  // Specific nf-core/sarek params
+  annotation_cache = true
+  pon = '/data1/PON/vcfs/BTB.PON.vcf.gz'
+  pon_index = '/data1/PON/vcfs/BTB.PON.vcf.gz.tbi'
+  snpEff_cache = '/data1/cache/snpEff/'
+  vep_cache = '/data1/cache/VEP/'
+}
+
+// Specific nf-core/sarek process configuration
+process {
+  withLabel:sentieon {
+    if (params.sentieon) {
+      module = 'sentieon/201808.05'
+      container = ''
```

Comment:
`''` not `null`?

Reply:
did not try with `null` actually, it was working with `''`.
But I can try that.

Reply:
Done, and it works